src/routes/VolunteerListTab/VolunteerListTab.js

```diff
@@ -77,25 +77,38 @@ export default class VolunteerListTab extends React.Component {
         this.setState((previousState)=>update(previousState,mergeValue));
     }
 
-    handleAddSingleVolunteer(profile_email, department, diff) {
-        let profile_id;
-        // TODO - create get request to test user validity
-        // axios.get('spark/user/' + profile_email)
-        // .then((res) => profile_id = res.data.id)
-        // .catch(this.logNetworkError);
-        console.log('got user info');
-        // add new volunteer
-        let query = Object.keys(diff).reduce((acc,cur) => 
-            acc + `&${cur}=${diff[cur]}`, '').replace('&','?');
+    handleAddVolunteers(profile_email, department, diff) {
         // TODO - convert department to department id
+        // TODO - create a request to test emails validity
+        if(profile_email.length < 1) {
+            console.log('no volunteers to add');
+            return;
+        }
+        const params = new URLSearchParams();
+        const query = profile_email.map((email) => {
```

Comment:
I agree, I read another article and I think there's another way - I'll try it later